.gitignore

```diff
@@ -8,8 +8,14 @@
 .sass-cache
 coverage
 Gemfile.lock
+Gemfile-local
```

Comment:
what is this file? 🤔

Reply:
allows to include dev dependencies that might not be relevant for the project or for other contributors, e.g.: `gem 'pry-debug'`